app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt

```diff
@@ -162,7 +122,18 @@ object TelemetryWrapper {
 
     @UiThread // via TelemetryHomeTileUniqueClickPerSessionCounter
     fun stopSession(context: Context) {
-        TelemetryHolder.get().recordSessionEnd()
+        fun safeEndSession() {
+            try {
+                TelemetryHolder.get().recordSessionEnd()
+            } catch (e: IllegalStateException) {
```

Comment:
If this exception is being thrown by the library, should we fix this in the library instead?

Reply:
Yeah, that's a good point.  I'll open an issue up, and start work on a PR.

Reply:
Opened issue https://github.com/mozilla-mobile/android-components/issues/822
and PR https://github.com/mozilla-mobile/android-components/pull/823